### PR TITLE
Migration of Extending section

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| Versions      | master

### CODE REVIEW

File: docs/extending/adding_tab_to_udw.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# Creating a UDW tab
```
Comment (DominikaK) [inline docs/extending/adding_tab_to_udw.md]: Why?

File: docs/extending/extending_date_and_time.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Format date and time
+
+## Twig filters
+
+Apart from changing the [date and time formats](../config_back_office#date-and-time-formats), you can use Twig filters:
```
Comment (DominikaK) [inline docs/extending/extending_date_and_time.md]:
```suggestion
Apart from changing the [date and time formats](../guide/config_back_office.md#date-and-time-formats), you can use Twig filters:
```

File: docs/releases/ez_platform_v2.0.0.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ eZ Platform has become a pure Symfony application, based on Symfony 3, which bri
 #### Back-office interface
 
 The back-office interface no longer uses YUI, and is instead based on React components and Bootstrap, which makes it easier to extend.
-See [the extensibility page](../guide/extending/extending_ez_platform.md#general-extensibility) to learn how to extend the new version of the UI.
+Explore Extending section in the menu to learn how to extend the new version of the UI.
```
Comment (DominikaK) [inline docs/releases/ez_platform_v2.0.0.md]:
```suggestion
Explore the Extending section in the menu to learn how to extend the new version of the UI.
```

File: mkdocs.yml [hunk 4]
```diff
@@ -192,29 +191,36 @@ nav:
...
+        - 'Extending Workflow': 'extending/extending_workflow.md'
+        - 'Extending Online Editor':
+            - 'Extending Online Editor': 'extending/extending_online_editor.md'
+            - 'Creating Online Editor plugin': 'extending/online_editor_plugin.md'
+        - 'Extending Modules':
+            - 'Extending UDW': 'extending/extending_udw.md'
+            - 'Creating a UDW tab': 'extending/adding_tab_to_udw.md'
+            - 'Extending Multi-file Upload': 'extending/extending_multifile_upload.md'
+            - 'Extending Sub-items List': 'extending/extending_subitems_list.md'
+            - 'Creating custom drop-downs': 'extending/creating_custom_dropdowns.md'
```
Comment (DominikaK) [inline mkdocs.yml]: Not really specific to modules, AFAIK, so better under Extending Back Office

File: docs/extending/adding_tab_to_udw.md (hunk 1 above)
Comment (MagdalenaZuba) [reply docs/extending/adding_tab_to_udw.md]: No idea, fixing it now

**Pull request merged**